test/core/get_local.wast

```diff
@@ -146,3 +146,21 @@
   "unknown local"
 )
 
+;; get_local should have retval
+(assert_invalid
```

Comment:
Nit: Can you move these tests up to the other "invalid typing" tests?

Reply:
Sure, done.